components/CompletionScoreCard.tsx

```diff
@@ -0,0 +1,213 @@
+  const [scores, setScores] = useState<
+    { status: 'High' | 'Medium' | 'Low'; score: string }[]
+  >([]);
+  const [completionPercentage, setCompletionPercentage] = useState<number>(0);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setData(mockData);
+      const calculatedScores = calculateScores(mockData.scoreData);
+      setScores(calculatedScores);
+      const percentage = calculatePercentage(mockData.scoreData);
+      setCompletionPercentage(percentage);
+    }, 1000);
+  }, []);
+
+if (!data) {
+  return <Typography>Loading...</Typography>;
+}
+
+  return (
+    <Box
+      sx={{
+        width: '80%',
+        margin: '0 auto',
+        marginTop: '7em',
+        border: '1px solid #ddd',
+        borderRadius: '0.5em',
+        padding: '1em',
+        backgroundColor: '#f9f9f9',
+        boxShadow: '0 4px 8px rgba(0, 0, 0, 0.1)',
+      }}
+    >
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          marginBottom: '1em',
+        }}
+      >
+        <Box>
+          <Typography variant="h5">{data.title}</Typography>
+          <Box
+            component="span"
+            sx={{
+              display: 'inline-flex',
+              alignItems: 'center',
+              textDecoration: 'none',
+              '&:hover': {
+                borderBottom: '1px solid',
+              },
+            }}
+          >
+            <Link
+              href={data.linkHref}
+              variant="body2"
+              sx={{
+                display: 'inline-flex',
+                alignItems: 'center',
+                textDecoration: 'none',
+              }}
+            >
+              {data.linkText}
+              <ArrowUpwardIcon
+                sx={{
+                  fontSize: '1.1em',
+                  transform: 'rotate(45deg)',
+                  marginLeft: '0.3em',
+                }}
+              />
+            </Link>
+          </Box>
+        </Box>
+        <Button
+          sx={{
+            backgroundColor: 'transparent',
+            color: 'primary.main',
+            '&:hover': {
+              backgroundColor: 'transparent',
+              color: 'primary.main',
+              border: '1px solid',
+              borderColor: 'brand.800',
+            },
+          }}
+          variant="contained"
+          color="primary"
+          href="/dashboard"
+        >
+          Explore city dashboard{' '}
+          <ArrowUpwardIcon
+            sx={{
+              transform: 'rotate(45deg)',
+              fontSize: '1.2em',
+              marginLeft: '0.5em',
+            }}
+          />
+        </Button>
+      </Box>
+      <Box
+        sx={{
+          width: '60%',
+          backgroundColor: '#f0f0f0',
+          padding: '1em',
+          borderRadius: '0.5em',
+        }}
+      >
+        <Box
+          sx={{ display: 'flex', alignItems: 'center', marginBottom: '0.5em' }}
```

Comment:
Rather than using `em` values, we can use numbers which will automatically use the correct spacing based on the default space size defined in the theme e.g. `marginBottom: 0.5`, `gap: 0.5`, `padding: 1`